### PR TITLE
note about selecting appropriate architecture

### PR DESCRIPTION
**Pull request by keichwa**
# Description

add a note about selecting the appropriate architecture
https://github.com/SUSE/spacewalk/issues/17459

Short summary of why you created this PR

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Comment by keichwa** [comment]
I added a general note at the beginning of the chapter.  Or must we add some such note to all the client registration sections?

**Comment by juliogonzalez** [comment]
> I added a general note at the beginning of the chapter. Or must we add some such note to all the client registration sections?

If you ask me, at the client registration sections as well, as a lot of people don't read the  start of the channel (because they use the left menu to browse).

In fact, I'd add it at the page for each client, not for each family.

For example at
https://www.uyuni-project.org/uyuni-docs/en/uyuni/client-configuration/clients-sle.html
not at
https://www.uyuni-project.org/uyuni-docs/en/uyuni/client-configuration/registration-overview-suse.html

File: modules/client-configuration/pages/registration-overview.adoc [hunk 1]
```diff
@@ -5,6 +5,12 @@ There are several ways to register clients to your {productname} Server.
 This section covers the various available methods.
 It also contains information specific to the operating system you intend to run on the client.
 
+[NOTE]
+====
+In the following section, descriptions often defaults to the [literal]``x86_64`` architecture.
+Replace it withother architectures if appropriate.
```
Comment (0rnela) [inline modules/client-configuration/pages/registration-overview.adoc:11]:
```suggestion
Replace it with other architectures if appropriate.
```

**Review by jcayouette — APPROVED** [review]

File: modules/client-configuration/pages/registration-overview.adoc (hunk 1 above)
Comment (keichwa) [reply modules/client-configuration/pages/registration-overview.adoc:11]: thanks for spotting!

**Comment by keichwa** [comment]
> In fact, I'd add it at the page for each client, not for each family.
> 
> For example at https://www.uyuni-project.org/uyuni-docs/en/uyuni/client-configuration/clients-sle.html not at https://www.uyuni-project.org/uyuni-docs/en/uyuni/client-configuration/registration-overview-suse.html

Yes, makes a lot of sense --Lana (quoting her teacher): "Every page is page one."

I'll try the snippet trick.

**Comment by keichwa** [comment]
[suse_manager_client-configuration_guide.pdf](https://github.com/uyuni-project/uyuni-docs/files/8556404/suse_manager_client-configuration_guide.pdf)

**Review by juliogonzalez — COMMENTED** [review]
I did a very, very light review.

If this snippet is added correctly at all clients:

image

Then the PR is OK. But please get some else from doc to check that the snippet is correctly added everywhere :-)

**Review by jcayouette — APPROVED** [review]

**Pull request merged**